src/Adapters/Files/FilesAdapter.js

```diff
@@ -8,12 +8,15 @@
 // * deleteFile(filename)
 // * getFileData(filename)
 // * getFileLocation(config, filename)
+// Adapter classes should implement the following functions:
+// * validateFilename(filename)
 //
 // Default is GridFSBucketAdapter, which requires mongo
 // and for the API server to be using the DatabaseController with Mongo
 // database adapter.
 
 import type { Config } from '../../Config';
+import Parse from 'parse/lib/node/Parse';
```

Comment:
import Parse from 'parse/node';

Its the same thing but simpler.

Reply:
👍